docs/mute.md

```diff
@@ -43,7 +43,20 @@ Note that the assertion is verified at the end of the program, so it doesn't sto
 You can add many ```TIMEOUT``` assertion in the same program, but, obviously, just the one with the shortest timeout is considered.
 
 There isn't a mandatory position to insert the ```TIMEOUT``` annotation line, that, as other ```MUTE``` annotations, are just considered as comments by the standard interpreter engine.
- 
+
+### Failing assertions 
+Like in other frameworks for unit testing, there is an assertion that always fails: 
+
+```
+    MU* FAIL('Something went wrong')
+```
+
+An expression can be used for the failing message:
+``` 
+      D MSG             S             15    INZ('Failure message')
+     MU* FAIL(MSG)
+ ```
+
```

Comment:
We could add an example to explain a case in which is useful. For example to indicate that a certain portion of the code is not expected to be executed.

Also, is there any limitation? Can the FAIL annotation be used anywhere or does it need to be put in front of a statement?

Reply:
Good point: I modified the docs